sdks/python/apache_beam/ml/inference/vertex_ai_inference.py

```diff
@@ -256,3 +257,46 @@ def validate_inference_args(self, inference_args: Optional[Dict[str, Any]]):
 
   def batch_elements_kwargs(self) -> Mapping[str, Any]:
     return self._batching_kwargs
+
+
+class VertexAITritonModelHandler(ModelHandler):
+    """
+    A custom model handler for Vertex AI endpoints hosting Triton Inference Servers.
+    It constructs a payload that Triton expects and calls the raw predict endpoint.
+    """
+    
+    def __init__(self, endpoint_url: str):
+        self.endpoint_url = endpoint_url
```

Comment:
Taking the single URL is somewhat unfriendly to the user when we can construct the endpoint string consistently with project ID, region, and endpoint names.